### DISCUSSION TITLE
Welcome to MyWhoosh2Garmin Discussions!

### DISCUSSION THREAD

**Discussion by JayQueue** [JayQueue/MyWhoosh2Garmin#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mateusperu** [comment]
hey @JayQueue , nice initiative.
I'm new to MyWhoosh (moving from Rouvy).
I tried to use your script but getting error trying to run pip install:

pip : The term 'pip' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path
was included, verify that the path is correct and try again.
At line:1 char:1
+ pip install pipenv

maybe I'm missing something ?
Thanks in advance.

**Reply by Wronixx** [reply]
I assume it's Windows, so most probably first of all you should install Python (if you haven't yet), download from python.org and install, then if pip is still not recognized as the cmdlet:

https://stackoverflow.com/questions/36835341/pip-is-not-recognized
